src/yunohost/backup.py

```diff
@@ -255,29 +274,66 @@ def _clean_tmp_dir(retcode=0):
         _clean_tmp_dir(1)
         raise MoulinetteError(errno.EINVAL, m18n.n('backup_nothings_done'))
 
+    # Add unlisted files from backup tmp dir
+    add_into_list(backup_csv_path,'backup.csv')
+    add_into_list(os.path.join(tmp_dir,'info.json'),'info.json')
+    if len(info['apps']) > 0:
+        add_into_list(os.path.join(tmp_dir,'apps'),'apps')
+    if os.path.isdir(os.path.join(tmp_dir,'conf')):
+        add_into_list(os.path.join(tmp_dir,'conf'),'conf')
+    if os.path.isdir(os.path.join(tmp_dir,'data')):
+        add_into_list(os.path.join(tmp_dir,'data'),'data')
+    backup_csv.close()
+    os.system("cp "+tmp_dir+"/backup.csv ~/")
     # Calculate total size
-    backup_size = int(subprocess.check_output(
-        ['du', '-sb', tmp_dir]).split()[0].decode('utf-8'))
+    backup_size = 0
+    with open(backup_csv_path, "r") as backup_csv:
+        for line in backup_csv:
+            line = line.rstrip('\n')
+            source, dest = line.split("\t", 1)
+            if source != os.path.join(tmp_dir,'info.json'):
+                backup_size += int(subprocess.check_output(
```

Comment:
We may want to use [https://docs.python.org/2/library/os.html#os.stat](os.stat) to get it a more pythonic and simple way :-)